index.html

```diff
@@ -9,11 +9,17 @@
 </head>
 
 <body>
-    <header class="container">
+    <header class="container" id="cont">
```

Comment:
- [ ] [REQUIRED] You did a great job on the header section 👍 , but your `navigation bar` elements are separated from the body elements, please fix this problem. I have `highlighted` the problem in the image below:

![image](https://user-images.githubusercontent.com/63915024/194590011-da1706c6-0b5f-4669-8475-24a257326373.png)

Reply:
@mukhammadrizooff please I haven't understood this point. Should I put the nav element out of the body?

Reply:
- [ ] Please try to fix the issue adressed by the previous reviewer. It seems like positioning of navbar is right. You can try to check 'headline' section for the problem : 
![image](https://user-images.githubusercontent.com/91486791/194619103-aa472442-d7df-400d-a592-b40fbe978eee.png)